pkg/logs/sender/config.go

```diff
@@ -89,3 +84,15 @@ func BuildEndpoints() (*client.Endpoints, error) {
 
 	return client.NewEndpoints(main, additionals), nil
 }
+
+func isSetAndNotEmpty(config config.Config, key string) bool {
+	return config.IsSet(key) && len(config.GetString(key)) > 0
+}
+
+//getLogsApiKey provides the dd api key used by the main logs agent sender.
+func getLogsAPIKey(config config.Config) string {
+	if isSetAndNotEmpty(config, "logs_config.logs_api_key") {
```

Comment:
I think you forgot to update `logs_api_key` to `api_key` here and bellow.
```suggestion
	if isSetAndNotEmpty(config, "logs_config.api_key") {
```